<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>817</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(90, 113, 94);
border:None;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">border:None;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
border:None;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QFrame" name="frame_7">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QPushButton" name="pushButton_14">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>81</width>
               <height>61</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">image: url(Media/Wix Logo Maker - Google Chrome 26-08-2022 16_51_23.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_8">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QPushButton" name="pushButton_2">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>10</y>
               <width>41</width>
               <height>41</height>
              </rect>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
image: url(Media/notification (2).png);
}

QPushButton:hover{
	image: url(Media/notification.png);
}

QPushButton:pressed{
	image: url(Media/notification (1).png);
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_6">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="Register_Button">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>0</y>
            <width>71</width>
            <height>71</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(97, 97, 97);
background-color: rgb(244, 249, 244);
border-radius:35px;</string>
          </property>
          <property name="text">
           <string>Register</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">/*background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 179, 255, 255), stop:1 rgba(255, 255, 255, 255));*/

background-color: rgb(138, 174, 146);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(204, 236, 211);
border-radius:15px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_8">
            <property name="font">
             <font>
              <family>Javanese Text</family>
              <pointsize>16</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	color: rgb(0, 0, 32);
	border-radius:10px;
	background-color: rgb(138, 174, 146);
}


</string>
            </property>
            <property name="text">
             <string>Home</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="About_button">
            <property name="font">
             <font>
              <family>Javanese Text</family>
              <pointsize>16</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
color: rgb(97, 97, 97);
border-radius:10px;
}

QPushButton:hover{
background-color: rgb(220, 255, 228);
	/*color: rgb(255, 255, 255);*/
	font: 700 16pt &quot;Constantia&quot;;
}
QPushButton:pressed{
	color: rgb(0, 0, 32);
	border-radius:10px;
	background-color: rgb(138, 174, 146);
}



</string>
            </property>
            <property name="text">
             <string>About</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Analytics_button">
            <property name="font">
             <font>
              <family>Javanese Text</family>
              <pointsize>15</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
color: rgb(97, 97, 97);
border-radius:10px;
}

QPushButton:hover{
background-color: rgb(220, 255, 228);
	/*color: rgb(255, 255, 255);*/
	font: 700 16pt &quot;Constantia&quot;;
}

QPushButton:pressed{
	color: rgb(0, 0, 32);
	border-radius:10px;
	background-color: rgb(138, 174, 146);
}


</string>
            </property>
            <property name="text">
             <string>Analytics</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Consult_button">
            <property name="font">
             <font>
              <family>Javanese Text</family>
              <pointsize>15</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
color: rgb(97, 97, 97);
border-radius:10px;
}

QPushButton:hover{
background-color: rgb(220, 255, 228);
	/*color: rgb(255, 255, 255);*/
	font: 700 16pt &quot;Constantia&quot;;
}

QPushButton:pressed{
	color: rgb(0, 0, 32);
	border-radius:10px;
	background-color: rgb(138, 174, 146);
}


</string>
            </property>
            <property name="text">
             <string>Consult</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Support_button">
            <property name="font">
             <font>
              <family>Javanese Text</family>
              <pointsize>15</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
color: rgb(97, 97, 97);
border-radius:10px;
}

QPushButton:hover{
background-color: rgb(220, 255, 228);
	/*color: rgb(255, 255, 255);*/
	font: 700 16pt &quot;Constantia&quot;;
}

QPushButton:pressed{
	color: rgb(0, 0, 32);
	border-radius:10px;
	background-color: rgb(138, 174, 146);
}


</string>
            </property>
            <property name="text">
             <string>Support</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Contact_button">
            <property name="font">
             <font>
              <family>Javanese Text</family>
              <pointsize>15</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
color: rgb(97, 97, 97);
border-radius:10px;
}

QPushButton:hover{
background-color: rgb(220, 255, 228);
	/*color: rgb(255, 255, 255);*/
	font: 700 16pt &quot;Constantia&quot;;
}

QPushButton:pressed{
	color: rgb(0, 0, 32);
	border-radius:10px;
	background-color: rgb(138, 174, 146);
}


</string>
            </property>
            <property name="text">
             <string>Contact</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="styleSheet">
          <string notr="true">background:transparent;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QFrame" name="frame_9">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QFrame" name="frame_11">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>100</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QTextBrowser" name="textBrowser">
                  <property name="html">
                   <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;Welcome to restorehealth&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_12">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QFrame" name="frame_15">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_9">
                   <item>
                    <widget class="QPushButton" name="Nutrition_button">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>180</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>180</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="cursor">
                      <cursorShape>PointingHandCursor</cursorShape>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color:rgb(97, 97, 97);
image: url(Media/Nutrition.jpeg);
background-color: rgb(244, 249, 244);</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_16">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_8">
                   <item>
                    <widget class="QPushButton" name="Mental_health_button">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>180</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>180</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="cursor">
                      <cursorShape>PointingHandCursor</cursorShape>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color:rgb(97, 97, 97);
image: url(Media/Mental_health.jpeg);
background-color: rgb(244, 249, 244);</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_13">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QFrame" name="frame_17">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_10">
                   <item>
                    <widget class="QPushButton" name="pushButton_12">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>180</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>180</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="cursor">
                      <cursorShape>PointingHandCursor</cursorShape>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color:rgb(97, 97, 97);
image: url(Media/Sleep.jpeg);
background-color: rgb(244, 249, 244);</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_18">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_11">
                   <item>
                    <widget class="QPushButton" name="pushButton_13">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>180</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>180</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="cursor">
                      <cursorShape>PointingHandCursor</cursorShape>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color:rgb(97, 97, 97);
image: url(Media/healh_fitness.jpeg);
background-color: rgb(244, 249, 244);</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_10">
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QScrollBar" name="verticalScrollBar">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>20</width>
               <height>501</height>
              </rect>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Media/home_page.qrc"/>
 </resources>
 <connections/>
</ui>
